go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/PaesslerAG/jsonpath v0.1.1
 	github.com/briandowns/spinner v1.18.1
+	github.com/cli/safeexec v1.0.0
```

Comment:
I don't see where this is being used

Reply:
yep, not used, was thinking to update our homebrew path check completely with what gh uses, but figured I might be lacking some context and reverted, forgot to remove this. Thanks, I'll remove